galley/pkg/config/resource/serialize.go

```diff
@@ -88,6 +88,10 @@ func Deserialize(e *mcp.Resource) (*Entry, error) {
 		return nil, fmt.Errorf("error unmarshaling create time: %v", err)
 	}
 
+	if err = types.UnmarshalAny(e.Body, p); err != nil {
```

Comment:
nit: maybe call the file `io.go`, since it supports both serialization and deserialization?

Reply:
io implies device interaction though. Named it "serialization" to make it a bit more neutral.